addons/web/static/src/scss/modal.scss

```diff
@@ -7,6 +7,10 @@
             margin-left: 10px;
         }
 
+        .modal-header .modal-title {
```

Comment:
Which use case do you intend to improve ? 

Because here you're modifying all the modals globally.

Reply:
This one: https://youtu.be/ZWv6kA9yRbo?t=68

When a string is too long for the modal title box, it overflows.

This was done regarding RTH review on 09/06/2020. See task pad for more infos.